### ISSUE TITLE
Notifications when service is up again

### ISSUE THREAD

**Issue by guillomep** [TwinProduction/gatus#9]
It would be great if we can add an alert when the service is up again.
We used it to monitor our kubernetes services and so it can be down for a time then up again without any actions, so if we can be inform that the service is up again it's great.

Something like that could be used

```
alerts:
      - type: slack
        enabled: true
        description: "healthcheck failed 3 times in a row"
        upAlert: true
```

**Comment by TwinProduction** [comment]
Just started working on this, shouldn't take long but here's a sneak peak:
![slack-alerts](https://user-images.githubusercontent.com/15699766/92293262-25aedd00-eef0-11ea-95b7-8f3e8ca57036.png)

What do you think?

**Comment by guillomep** [comment]
Yeah !

**Comment by TwinProduction** [comment]
No problem :) 

FYI, this has been released in v0.1.2
